1906-03-26.xml

```diff
@@ -0,0 +1,5314 @@
+                                <cell>4 p.m.</cell>
+                                <cell><name>S.S. Semiramis</name></cell>
+                                <cell><persName>Capt. Martinsolich</persName></cell>
+                                <cell><date when="1906-11-17">March 21</date></cell>
+                                <cell>4 p.m.</cell>
+                                <cell><name>S.S. Semiramis</name></cell>
+                                <cell><persName>Capt. Martinelich</persName></cell>
+                            </row>
+                        </table>
+                        <p>Fortnightly Service: Alexandria-Brindisi &amp; Trieste Line.</p>
+                        <p>Steamer leaves Alexandria on or about 14 March, 1, 11,and 25 April; 1, 9,
+                            and 22 May; 6 June, 4 p.m.</p>
+                        <p>Syrian-Cyprus-Caramanian Line.</p>
+                        <p>Steamer leaves Alexandria 5 and 19 March.</p>
+                        <p>Syrian-Caramanian Line.</p>
+                        <p>Steamer leaves Alexandria on or about 12 and 26 March; 9 and 23 April; 7
+                            and 21 May; 4 and 18 June.</p>
+                        <p>Far East Lines.</p>
+                        <p>Departures from Port Said: To Suez, Aden, Bombay, Colombo, Penang,
+                            Singapore, Hong-Kong, Shanghai, Yokohama and Kobé, about <date
+                                when="1906-10-04">5 March</date>, <date when="1905-11-03">3
+                                April</date>,<date when="1905-11-03">4 May</date>, <date
+                                when="1905-11-03">3 June</date>, <date when="1905-11-03">4
+                                July</date>, <date when="1905-11-03">3 August</date>. To Suez, Aden
+                            and Bombay accelerated service about 8 and 17 March, 8 April, 8 May, 17
+                            June. To Suez, Aden, Karachi, Colombo, Madras Rangoon and Calcutta about
+                            17 March, 19 April, 19 May. To Suez, Aden, Karachi and Bombay about 13
+                            March, 13 April, 13 November, 13 December (Winter Lane).</p>
+                        <p>East African Line.</p>
+                        <p>Departures from Port Said: To Suez, Aden, Mombassa, Zanzibar, Beira,
+                            Delagoa-Bay, and Durban about <date when="1906-10-03">5 March</date>,
+                                <date when="1905-11-02">2 April</date>, <date when="1905-12-03">3
+                                May</date>, <date when="1905-12-03">2 June</date>, <date
+                                when="1905-12-03">3 July</date>, <date when="1905-12-03">2
+                                Aug.</date>, <date when="1905-12-03">2 Sept.</date>, <date
+                                when="1905-12-03">3 Oct.</date>, <date when="1905-12-03">2
+        
```

Comment:
make the headlines in each local and general item a `<head>` rather than a `<p>`